Test/TestMzML.cs

```diff
@@ -1563,6 +1563,15 @@ public static void TestDynamicMzmlWithPeakFiltering()
             }
         }
 
+        [Test]
+        public static void TestEthcdReading()
+        {
+            string filePath = Path.Combine(TestContext.CurrentContext.TestDirectory, "DataFiles", "sliced_ethcd.mzML");
+            Mzml mzml = Mzml.LoadAllStaticData(filePath, null, 1);
+            var ethcdScan = mzml.GetOneBasedScan(6);
+            Assert.That(ethcdScan.DissociationType == DissociationType.EThcD);
+        }
+
```

Comment:
I do feel like we need unit tests for "dissociationType = scanDissociationTypes.First();" and "dissociationType = DissociationType.Unknown;"

Reply:
I don't know how to write one for DissociationType.Unknown. This would mean that the scan has a dissociation type that we don't know how to interpret. I don't know if that exists

Reply:
can you edit your mzml slide to have dissociation type xyz which would just be a place holder for any that get added in the future?

I'm not hung up on this if its a pain. I just like high unit test coverage.

Reply:
ok, done